docs/standard/linq/retrieve-paragraphs-office-open-xml-document.md

```diff
@@ -214,7 +214,7 @@ Module Module1
         Return (e.<w:r>.<w:t>).StringConcatenate(Function(element) CStr(element))
     End Function
 
-    ' Following function is required because Visual Basic does not support short circuit evaluation
+    ' Following function is required because Visual Basic doesn't support short circuit evaluation
```

Comment:
```suggestion
    ' The following function is required because Visual Basic doesn't support short circuit evaluation
```

Reply:
I haven't been editing the code examples or their comments at all, though sometimes global search and replace can change things.